CONTRIBUTING.md

```diff
@@ -87,6 +87,23 @@ Then, pass this device to Dradis and select a test configuration file:
 $ cargo run --bin dradis -- --device /dev/media0 ./dradis/samples/test-single-mode-720p.yaml
 ```
 
+## Development environment
+
+You can use any text editor or IDE. But, if you are using Visual Studio
+Code, the recommended extensions to install are:
+
+- [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer):
+  Rust language support with code completion, diagnostics, refactoring and more
+- [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb):
+  Native debugger for Rust (used in the debug configuration)
+- [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig):
+  Configure vscode using the .editorconfig file
+- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker):
+  Install this extension to avoid typos
```

Comment:
I'm really not into recommending third-party extensions I know nothing about, even more so when rustfmt is already there for some of them, and already has integration in vscode.